lucet-runtime/lucet-runtime-tests/src/stack.rs

```diff
@@ -1,20 +1,16 @@
 use failure::Error;
 use lucet_runtime_internals::module::DlModule;
 use lucetc::Lucetc;
-use std::fs::File;
-use std::io::prelude::*;
 use std::sync::Arc;
 use tempfile::TempDir;
+use wabt::wat2wasm;
 
 pub fn stack_testcase(num_locals: usize) -> Result<Arc<DlModule>, Error> {
-    let workdir = TempDir::new().expect("create working directory");
-
-    let wasm_path = workdir.path().join("out.wasm");
+    let wasm_program = wat2wasm(generate_test_wat(num_locals)).expect("test wat is valid");
```

Comment:
Since `Lucetc` takes care of `wat2wasm` for the "from a file" case, I think it makes for a less surprising API for it to handle that translation in the "from bytes" case as well. Can you adjust `from_bytes` to also do a preamble check/translation? Might make sense to just split `lucetc/src/load.rs:read_module` up a bit for that

Reply:
ooh, good idea! i can do that